workbench-core/authorization/src/dynamicAuthorization/wbcGroupManagementPlugin.test.ts

```diff
@@ -170,15 +157,53 @@ describe('WBCGroupManagemntPlugin', () => {
     });
   });
 
-  describe('setGroupStatus', () => {
-    let groupId: string;
-    let status: GroupStatus;
+  describe('removeUserFromGroup', () => {
+    it('returns the groupId and userId in the data object when the user was successfully removed', async () => {
+      const response = await wbcGroupManagementPlugin.removeUserFromGroup({
+        groupId,
+        userId,
+        authenticatedUser: mockUser
+      });
+
+      expect(response).toMatchObject<RemoveUserFromGroupResponse>({ data: { groupId, userId } });
+    });
+
+    it('throws IdpUnavailableError when the IdP encounters an error', async () => {
+      mockUserManagementPlugin.removeUserFromRole = jest.fn().mockRejectedValue(new IdpUnavailableError());
+
+      await expect(
+        wbcGroupManagementPlugin.removeUserFromGroup({ groupId, userId, authenticatedUser: mockUser })
+      ).rejects.toThrow(IdpUnavailableError);
+    });
+
+    it('throws PluginConfigurationError when the UserManagementService has a configuration error', async () => {
+      mockUserManagementPlugin.removeUserFromRole = jest
+        .fn()
+        .mockRejectedValue(new PluginConfigurationError());
 
-    beforeEach(() => {
-      groupId = 'groupId';
-      status = 'active';
+      await expect(
+        wbcGroupManagementPlugin.removeUserFromGroup({ groupId, userId, authenticatedUser: mockUser })
+      ).rejects.toThrow(PluginConfigurationError);
+    });
+
+    it('throws GroupNotFoundError when the group already exists', async () => {
```

Comment:
is it meant to be 'when the group does not exist'?

Reply:
copy paste error. Fixed